### PR TITLE
ROX-16874: Add notifiers to declarative config E2E tests

### PR DESCRIPTION
**Pull request by dhaus67**
## Description

This PR adds notifiers to the E2E tests of declarative configuration.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

- see CI passing.

### CODE REVIEW

**Comment by dhaus67** [comment]
Current dependencies on/for this PR:
* master
  * **PR #6304** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/6304" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/6304?utm_source=stack-comment).

File: qa-tests-backend/src/test/groovy/DeclarativeConfigTest.groovy [hunk 1]
```diff
@@ -442,6 +492,13 @@ oidc:
                 )
                 .getAuthProvidersCount() == 0
 
+        // No notifier should be created.
+        def notifierAfterDeletion = NotifierService.getNotifierClient().getNotifiers(
```
Comment (ivan-degtiarenko) [inline qa-tests-backend/src/test/groovy/DeclarativeConfigTest.groovy:496]: Hmm, why all results here are named `...AfterDeletion` when in reality they weren't created yet?

**Review by ivan-degtiarenko — APPROVED** [review]
One nit, otherwise LGTM

**Pull request merged**